R/expr__expr.R

```diff
@@ -1001,7 +1034,7 @@ Expr_rpow = function(base) {
 
 #' @rdname Expr_rpow
 #' @export
-"%**%.Expr" = function(e1, e2) e1$rpow(e2)
+"%**%.Expr" = function(e1, e2) result(wrap_e(e1)$rpow(e2)) |> unwrap("using the '**'-operator")
```

Comment:
What is the equivalent of this `rpow` in R?  

`**` is the same as `^` so putting `**` in the error message doesn't seem correct

Reply:
[oh I see py-polars does not document dunder method __pow__](https://github.com/pola-rs/polars/blob/d9ff23c5316ad489de9e4411c5e3dc275a214e71/py-polars/polars/expr/expr.py#L211)

I should remove it. It is only needed if rhs inherits `Expr` but not left. R does that automatically

Reply:
can you also remove the related TODO I added in the tests?

Reply:
rpow is now removed completely